integration-tests/src/constants.rs

```diff
@@ -331,6 +333,45 @@ pub mod politest {
 	const GENESIS_PARACHAIN_BOND_RESERVE_PERCENT: Percent = Percent::from_percent(0);
 	const GENESIS_NUM_SELECTED_CANDIDATES: u32 = 5;
 
+	pub fn set_prices() {
```

Comment:
Could make it a general function that you pass the PolitestNet or PolimecNet to, so we don't have it copy pasted. See how we generate the PolimecOrigin and PolimecRuntime types from PolimecNet to see how we could generalize

Reply:
We will remove the politest runtime soon, so we can keep it as is for now